### PR TITLE
auth docs: add note about EL9 and SHA1

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/upgrading.rst [hunk 1]
```diff
@@ -11,6 +11,11 @@ upgrade notes if your version is older than 3.4.2.
 any version to 4.8.x
 --------------------
 
+Use of SHA1 on Enterprise Linux 9 and derivatives
```
Comment (zeha) [inline docs/upgrading.rst:14]: should this header more clearly mention Red Hat?

Comment (kpfleming) [reply docs/upgrading.rst:14]: It probably should; if 'derivatives' are mentioned explicitly, then 'EL9' doesn't exist, it's 'RHEL9'.

Comment (Habbie) [reply docs/upgrading.rst]: adjusted

**Comment by omoerbeek** [comment]
The rec part has been handled in code and a mention in the upgrade notes.

**Comment by Habbie** [comment]
removed the rec doc bits from here

**Pull request merged**